### ISSUE TITLE
Invalid zone directive makes zone ineditable

### ISSUE THREAD

**Issue by mimi89999** [KaneRoot/dnsmanager#21]
Hello,
If I make a mistake in a zone directive (like add `$ORIGIN` without adding a domain afterwards), the interface will show an error like `Oh ! Une erreur est survenue. $ORIGIN incomplete at /home/ppittoli/perl5/lib/perl5/Net/DNS/ZoneFile.pm line 470, line 11. file /tmp/zones//noinidonoidewoindnoiew.netlib.re line 11 at /home/ppittoli/dnsmanager/bin/../lib/zonefile.pm line 89.` and it will be impossible to edit the zone.

**Comment by KaneRoot** [comment]
Did you go to IRC to talk about this problem?

I removed the file so you should be able to modify your zone again.

**Comment by mimi89999** [comment]
Yes. It was me. Thanks for removing the file. However, I think that it would be better to resolve the issue on the code.

Also, I noticed that https://git.karchnu.fr/ is down.

**Comment by KaneRoot** [comment]
> Yes. It was me. Thanks for removing the file. However, I think that it would be better to resolve the issue on the code.

Yep, but won't happen. I rewrote most of the code in a completely different way, so I won't change anything in the current code. I don't know when I will make the final swap.

> Also, I noticed that https://git.karchnu.fr/ is down.

Yep, the princess is in another castle Mario.
Link changed, thanks for the warning.
